datalad/tests/test_customremotes_archive.py

```diff
@@ -13,6 +13,8 @@
 from ..customremotes.base import AnnexExchangeProtocol
 from ..customremotes.archive import AnnexArchiveCustomRemote
 from ..cmd import Runner
+from ..support.gitrepo import GitRepo
+from ..support.dataset import Dataset as Handle
```

Comment:
It does indeed!

2015-04-15 15:28 GMT+02:00 Yaroslav Halchenko <notifications@github.com>:

> In datalad/tests/test_customremotes_archive.py
> <https://github.com/yarikoptic/datalad/pull/2#discussion_r28418544>:
>
> > @@ -13,6 +13,8 @@
> >  from ..customremotes.base import AnnexExchangeProtocol
> >  from ..customremotes.archive import AnnexArchiveCustomRemote
> >  from ..cmd import Runner
> > +from ..support.gitrepo import GitRepo
> > +from ..support.dataset import Dataset as Handle
>
> which suggests that we might want to rename Dataset to Handle?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yarikoptic/datalad/pull/2/files#r28418544>.
>